src/Compilers/CSharp/Portable/Symbols/SubstitutedMethodSymbol.cs

```diff
@@ -38,9 +38,10 @@ internal class SubstitutedMethodSymbol : WrappedMethodSymbol
 
         private int _hashCode; // computed on demand
 
-        internal SubstitutedMethodSymbol(SubstitutedNamedTypeSymbol containingSymbol, MethodSymbol originalDefinition)
+        internal SubstitutedMethodSymbol(NamedTypeSymbol containingSymbol, MethodSymbol originalDefinition)
             : this(containingSymbol, containingSymbol.TypeSubstitution, originalDefinition, constructedFrom: null)
         {
+            Debug.Assert(containingSymbol is SubstitutedNamedTypeSymbol || containingSymbol is SubstitutedErrorTypeSymbol);
```

Comment:
>containingSymbol is SubstitutedErrorTypeSymbol [](start = 75, length = 46)

Error types do not have members, is this yet another case where we are fed an error method symbol and try use it as if it is good? If that is the case, I would prefer us to fix the root cause of the problem, i.e. the problem of failing to recognize the error situation in the local rewriter, rather than fixing a symptom.